.github/workflows/main.yml

```diff
@@ -120,6 +120,7 @@ jobs:
         uses: ./.github/actions/setup
         with:
           clippy: true
+          cli: true
```

Comment:
Why are the CI deps needed now?

Reply:
Need libudev installed for the Rust client tests https://github.com/solana-program/token-2022/actions/runs/13358110458/job/37303700782